pkg/satellite/database.go

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellite
+
+// DB used to manage db connections and context through different repositories
+type DB interface {
+	User() Users
```

Comment:
`Users() Users`

Reply:
Also add comment to each method in interface, this helps readers understand things better.